schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -810,7 +810,13 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/Microsoft.Authorization.json#/resourceDefinitions/locks" },
                   { "$ref": "https://schema.management.azure.com/schemas/2014-10-01-preview/Microsoft.Authorization.json#/resourceDefinitions/roleAssignments" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Authorization.json#/resourceDefinitions/roleAssignments" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Authorization.json#/resourceDefinitions/roleDefinitions" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Authorization.json#/resourceDefinitions/roleDefinitions" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Authorization.json#/resourceDefinitions/policyAssignments" },
```

Comment:
>policyAssignments [](start = 134, length = 17)

have you tried to deploy each of these resource types from a template deployment? I'm not sure if they all work.

Reply:
No I haven't. It's not a scenario for policy which is a management technology. There is no realistic scenario in which a customer wants to create policy using deployment templates. Should I remove them?